### DISCUSSION TITLE
Pulse Guide Command To Mount has Failed

### DISCUSSION THREAD

**Discussion by stev0aus** [rmorgan001/GSServer#23] [Q&A] [answered]
Hi,

I am just in the process of switching over to GSS and it is amazing. Love the interface!

When setting up w/PHD2, I am recieving an error which is "pule guide command to mount has failed'. I have looked at all of my settings and cannot seem to work it out. Changed cables, two different PC's, no issues with power, reset, new pofile load with wizard etc. What am I missing? Any thoughts?

Thanks

Steve

**Comment by rmorgan001** [comment, marked as answer]
Review the documents\session log for errors and post to the https://gss.groups.io/g/main for support
